src/components/CreationGoveranceAction/Step2.jsx

```diff
@@ -475,7 +475,7 @@ const Step2 = ({
                                 label='Amount'
                                 type='tel'
                                 variant='outlined'
-                                placeholder='e.g. 2000'
+                                placeholder='e.g. 2000 ada'
```

Comment:
**Question**

Should this be in Ada or Lovelace? (I think Ada is fine for the frontend)
Which unit is the Wallet API expecting when creating the treasury withdrawal action?

@MSzalowski @vpaunovic

Reply:
@Ryun1 I approved the PR and pushed a commit with those max lengths for abstract, rationale, and motivation fields. Do you think we should wait for @MSzalowski's reply?

Reply:
It should be ADA. I don't know if there are any amounts presented in Lovelace expected by wallet API

Reply:
But not 100% sure to be honest :sweat_smile:

Reply:
lets do ada ! and hope make a note to recheck later